addons/website/models/mixins.py

```diff
@@ -85,11 +89,12 @@ def get_website_meta(self):
         if self.website_meta_description:
             opengraph_meta['og:description'] = self.website_meta_description
             twitter_meta['twitter:description'] = self.website_meta_description
-        meta_image = self.website_meta_og_img or opengraph_meta['og:image']
-        if meta_image.startswith('/'):
-            meta_image = "%s%s" % (root_url, meta_image)
-        opengraph_meta['og:image'] = meta_image
-        twitter_meta['twitter:image'] = meta_image
+        meta_image = self.website_meta_og_img
```

Comment:
`opengraph_meta['og:image']` also need to be processed to add the root_url and make it absolute link

![image](https://user-images.githubusercontent.com/30048408/173810680-9abee27a-08bd-4ccc-ad5e-cd76afaa4f28.png)